core/src/html/layout.rs

```diff
@@ -862,9 +862,9 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
     fn left_alignment_offset(span: &TextSpan, is_first_line: bool) -> Twips {
         if span.bullet {
             if is_first_line {
-                Twips::from_pixels(35.0 + span.left_margin + span.block_indent + span.indent)
+                Twips::from_pixels(36.0 + span.left_margin + span.block_indent + span.indent)
```

Comment:
Was this just to make the tests pass better? The change is kinda out of nowhere ;D

Reply:
Yes, I added a test that verifies how the bullet behaves when there's no font (to make sure I'm not breaking anything), and it turned out the bullet was misaligned by one pixel :sweat_smile: